tests/models/spatialview_model_tests.py

```diff
@@ -192,20 +193,35 @@ def test_create_spatialview_mixed_geometry(self):
 
         spatialview.delete()
 
+    def test_restore_state_from_serialized_graph(self):
+        spatialview = self.generate_valid_spatialview()
+        spatialview.full_clean()
+        spatialview.save()
+
+        graph = Graph.objects.get(pk=spatialview.geometrynode.graph.pk)
+        draft_graph = graph.create_draft_graph()
+
+        # updating graph from draft graph removes all elements
+        # including the serialized graph - then recreates them
+        graph.update_from_draft_graph(draft_graph=draft_graph)
```

Comment:
If we don't publish() here, then clean() crashes with `AttributeError: 'NoneType' object has no attribute 'publishedgraph_set'`. 

I think we should publish() to model correct usage.

Reply:
More context: what I was doing was fiddling with simplifying the next assertion to just `spatialview.refresh_from_db()`, which will raise if it's been deleted. I then added `full_clean()` right after and saw that it fails. This might be a nice update to the test.